src/Azure.Functions.Cli/StaticResources/stacks.json

```diff
@@ -0,0 +1,1789 @@
+{
+  "value": [
```

Comment:
Why are we storing a static version of the current stacks API response? Seems pretty unscalable to me as you would have to keep updating this file every time there are changes to the response.